### PR TITLE
Issue #36 - Add autosubmit for login form

### PR DESCRIPTION
**Pull request by thekarinka**

### CODE REVIEW

File: django_fido/js-tests/fido2.test.js [hunk 1]
```diff
@@ -54,7 +54,8 @@ describe('Fido 2', () => {
         document.getElementsByTagName('body')[0].appendChild(submit_button)
 
         await startFido2()
-        await submit_button.click()
+        if (mode !== 'authentication') // because there is autosubmit
```
Comment (stinovlas) [inline django_fido/js-tests/fido2.test.js]: There is a constant `FIDO2_AUTHENTICATION_REQUEST` in `fido2.js`.

File: django_fido/js/fido2.js [hunk 2]
```diff
@@ -124,18 +124,20 @@ async function startFido2() {
...
             e.preventDefault()
 
-            if (!isFido2Availabile()) {
-                addFido2Error(TRANSLATIONS.FIDO2_NOT_AVAILABLE)
-                return
-            }
-            if (form.dataset.mode === FIDO2_AUTHENTICATION_REQUEST) {
-                await sendFido2AuthenticationRequest(form.dataset.url)
-            } else if (form.dataset.mode === FIDO2_REGISTRATION_REQUEST) {
+            if (form.dataset.mode === FIDO2_REGISTRATION_REQUEST) {
```
Comment (stinovlas) [inline django_fido/js/fido2.js]: This condition could be much higher now, above the submit button detection.

Comment (thekarinka) [reply django_fido/js/fido2.js]: Yes, it will be better. Fixed.

Comment (stinovlas) [reply django_fido/js/fido2.js]: ok

**Pull request merged**